packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/eoobsv/use-flow.md

```diff
@@ -0,0 +1,94 @@
+The following sections outline the requests required to implement the out-of-band (OOB) flow (with SMS or Voice) using direct calls to the Okta OpenID Connect & OAuth 2.0 API.
+
+### Request for out-of-band authentication
+
+Before you can begin this flow, collect the username from the user in a manner of your choosing. Then, make an API call to the Okta [authorization server](/docs/concepts/auth-servers/) `/oob-authenticate` endpoint. Use this endpoint to initiate an authentication flow with an out-of-band factor (such as SMS or Voice) as the primary factor.
+
+> **Note:** The `/oob-authenticate` endpoint doesn't support multifactor authentication.
+
+Your request should look something like this:
+
+```bash
+curl --request POST \
+  --url https://${yourOktaDomain}/oauth2/v1/oob-authenticate \
+  --header 'accept: application/json' \
+  --header 'content-type: application/x-www-form-urlencoded' \
+  --data 'client_id=${client_id}&login_hint=${testuser%40example.com}&channel_hint=${sms or voice}'
+```
+
+Note the parameters that are passed:
+
+- `client_id`: Matches the client ID of the application that you created in the [Set up your app](#set-up-your-app) section. You can find it at the top of your app's **General** tab.
+- `login_hint`: The username (email) of a user registered with Okta
+- `channel_hint`: The out-of-band channel that the client wants to use. For example, `sms` or `voice`.
+
+For more information on these parameters, see the `/oob-authenticate` [endpoint](https://developer.okta.com/docs/api/openapi/okta-oauth/oauth/tag/OrgAS/#tag/OrgAS/operation/oob-authenticate).
+
+### OOB response example
+
+In an HTTP 200 response, Okta returns the following parameters:
+
+> **Note:** Okta also sends a push notification to the user.
+
+**SMS**
+
+```json
+  {
+      "oob_code": "ftwG4adUWWIV_hJi3OjYeE7PlIVia-aN3B",
+      "expires_in": 300,
+      "channel": "sms",
+      "binding_method": "prompt"
+  }
+```
+
+**Voice**
+
+```json
+  {
+      "oob_code": "ftpvP1LB26vCARL7EWM55cUhPA2vdQmHFp",
+      "expires_in": 300,
+      "channel": "voice",
+      "binding_method": "prompt",
+  }
+```
+
+Note the parameters included:
+
+- `oob_code`: An identifier of a single out-of-band factor transaction. To respond to or check on the status of an out-of-band factor, use this code to identify the factor transaction.
+- `expires_in`: The time, in seconds, until the `oob_code` expires
+- `channel`: The type of out-of-band channel used.
+- `binding_method`: The method used to bind the out-of-band channel with the primary channel. Supported value for `sms` or `voice` is `prompt`.
```

Comment:
"Supported value for `sms` or `voice` is `prompt`" -> "Supported value when `channel` is `sms` or `voice` is `prompt`"